worldengine/cli/main.py

```diff
@@ -392,9 +391,10 @@ def main():
 
     if args.seed is not None:
         seed = int(args.seed)
+        assert 0 <= seed <= 65535, "Seed has to be in the range between 0 and 65535, borders included."  # there is a hard limit somewhere so seeds outside the uint16 range are considered unsafe
```

Comment:
What exactly are you suggesting? A global variable? A local one? Something else?